app/controllers/vm_common.rb

```diff
@@ -539,7 +539,7 @@ def policy_options
 
   # Set right_size selected db records
   def right_size(record = nil)
-    @record ||= record ? record : find_records_with_rbac(params[:id]).first
+    @record ||= record ? record : find_record_with_rbac(params[:id])
```

Comment:
@romanblanco There should be `Vm` because you can't do Right-size recommendation on `Template`.